pkg/controller/common/common_reconciler.go

```diff
@@ -103,33 +107,20 @@ func (r *CommonReconciler) ConnectToContainerJFR(ctx context.Context, namespace
 	return jfrClient, nil
 }
 
-func (r *CommonReconciler) ConnectToPod(targetPod *corev1.Pod, jmxPort int32) error {
+// GetPodTarget returns a TargetAddress for a particular pod and port number
+func (r *CommonReconciler) GetPodTarget(targetPod *corev1.Pod, jmxPort int32) (*jfrclient.TargetAddress, error) {
 	// Have Container JFR connect to the target JVM
```

Comment:
This comment looks like it's outdated too

Reply:
Good catch, fixed this.